project/src/MainMenu/SettingsScreen.gd

```diff
@@ -2,12 +2,42 @@ extends MarginContainer
 
 onready var master_volume_slider : HSlider = $VBoxContainer/MasterVolume/MasterVolumeSlider
 onready var audio_effects_slider : HSlider = $VBoxContainer/SoundEffects/SoundEffectsSlider
+onready var language_en_button : TextureButton = $VBoxContainer/Language/English
+onready var language_pl_button : TextureButton = $VBoxContainer/Language/Polish
 
 func _ready():
+	
+	_set_values()
+	_connect_signals()
+
+
+func _set_values() -> void:
 	master_volume_slider.value = Settings.master_volume
 	audio_effects_slider.value = Settings.audio_effects_volume
-	
+
+	_set_language(-1)
```

Comment:
Nie dokońca rozumiem co tutaj znaczy -1?

Reply:
-1 znaczy że nie chcemy wywoływać Settings.set_language() w funkcji _set_language().
W tym miejscu to wywołanie byłoby bez sensu bo język się nie zmienia

Reply:
Skoro nie chcemy zmieniać języka to czemu wywołujemy te metode?

Reply:
Żeby nie powielać tego `match Settings.language:`. Wydaje mi się że tak kod wygląda ładniej.

Reply:
Ah widzę, to wystarczy że wydzielisz match do osobnej metody i wywołasz ją osobno w _set_language i osobno w _set_values.